src/main/java/jalau/at18/katas/highestlowest/rodrigov/Main.java

```diff
@@ -1,20 +1,22 @@
 package jalau.at18.katas.highestlowest.rodrigov;
+import javax.swing.JOptionPane;
 public class Main {
     public static void main(String[] args) {
-        final int thirdNumber = -3;
-        final int fourthNumber = 4;
-        final int fifthNumber = 5;
-        int[] numbers = {1, 2, thirdNumber, fourthNumber, fifthNumber};
-        int lowestNumber = numbers[0];
-        int highestNumber = numbers[0];
+        String numbersList = JOptionPane.showInputDialog("Input numbers:");
```

Comment:
This was not actually necessary. In fact, by adding unit tests, we could remove `main()` method because implementation code is being verified by the tests

Reply:
I fixed all this part on my exercises.